src/java.base/share/classes/javax/net/ssl/SSLSocketFactory.java

```diff
@@ -88,18 +88,15 @@ public static SocketFactory getDefault() {
 
     @SuppressWarnings("removal")
     static String getSecurityProperty(final String name) {
-        return AccessController.doPrivileged(new PrivilegedAction<>() {
-            @Override
-            public String run() {
-                String s = java.security.Security.getProperty(name);
-                if (s != null) {
-                    s = s.trim();
-                    if (s.isEmpty()) {
-                        s = null;
-                    }
+        return AccessController.doPrivileged((PrivilegedAction<String>) () -> {
+            String s = Security.getProperty(name);
+            if (s != null) {
+                s = s.trim();
+                if (s.isEmpty()) {
+                    s = null;
```

Comment:
According to the implementation of `Security::getProperty`. The return value is already trimmed.

Reply:
Removed those lines.